x-pack/plugins/ml/public/application/components/help_menu/index.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useEffect } from 'react';
+import { i18n } from '@kbn/i18n';
+import { useKibana } from '../../../../../../../src/plugins/kibana_react/public';
+
+export const HelpMenu = React.memo(() => {
+  const { chrome, docLinks } = useKibana().services;
+
+  useEffect(() => {
+    chrome.setHelpExtension({
+      appName: i18n.translate('xpack.ml.chrome.help.appName', {
+        defaultMessage: 'Machine learning',
+      }),
+      links: [
+        {
+          content: i18n.translate('xpack.ml.chrome.helpMenu.documentation', {
+            defaultMessage: 'Documentation',
+          }),
+          href: docLinks.links.ml.guide,
```

Comment:
Do we always want to link to the top level ML guide from every page in the ML app, or should the `href` be passed in to the `HelpMenu` component?

Reply:
Yes, I was thinking it would be nice to have that functionality, but wasn't sure of the best way to accomplish that. I'll maybe reach out to pick your brain on the right code to accomplish it.